subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/java/JavaIDEModelPerformanceTest.groovy

```diff
@@ -138,20 +129,20 @@ class JavaIDEModelPerformanceTest extends AbstractCrossVersionPerformanceTest {
 
         then:
         result.assertCurrentVersionHasNotRegressed()
-
-        where:
-        testProject                   | iterations
-        LARGE_MONOLITHIC_JAVA_PROJECT | 200
-        LARGE_JAVA_MULTI_PROJECT      | 40
     }
 
-    private setupTestProject(JavaTestProject testProject, int iterations) {
-        runner.testProject = testProject
-        runner.gradleOpts = ["-Xms${testProject.daemonMemory}", "-Xmx${testProject.daemonMemory}"]
+    private setupTestProject() {
+        runner.gradleOpts = runner.projectMemoryOptions
+        def iterations = determineIterations()
         runner.warmUpRuns = iterations
         runner.runs = iterations
     }
```

Comment:
This sets up the runner, or the iterations, not the test project.

Reply:
Renamed to `setupRunner()`.